src/SipHash.chpl

```diff
@@ -69,20 +81,32 @@ module SipHash {
     return c;
   }
   
-  proc sipHash64(msg: [] uint(8), D): uint(64) {
+  proc sipHash64(msg: [] ?t, D): uint(64) {
     var (res,_) = computeSipHashLocalized(msg, D, 8);
     return res;
   }
 
-  proc sipHash128(msg: [] uint(8), D): 2*uint(64) {
+  proc sipHash128(msg: [] ?t, D): 2*uint(64) {
```

Comment:
Throw a where clause on these to limit the types of `t` just to avoid adding any unexpected cases without checking the code in the future?

Reply:
Probably a good idea -- will add tomorrow.